doc/htconfig.md

```diff
@@ -58,9 +58,12 @@ Example: To set the automatic database cleanup process add this line to your .ht
 * **max_processes_backend** - Maximum number of concurrent database processes for background tasks. Default value is 5.
 * **max_processes_frontend** - Maximum number of concurrent database processes for foreground tasks. Default value is 20.
 * **min_poll_interval** - minimal distance in minutes between two polls for a contact. Default is 1. Reasonable values are between 1 and 59.
-* **memcache** (Boolean) - Use memcache. To use memcache the PECL extension "memcache" has to be installed and activated.
-* **memcache_host** - Hostname of the memcache daemon. Default is '127.0.0.1'.
-* **memcache_port** - Portnumber of the memcache daemon. Default is 11211.
+* **session_handler** ([database]|cache|native) - Whether to use Cache to store session data or to use PHP native session storage
```

Comment:
do the square bracket indicate a default value?

Reply:
Indeed, I wanted to use bold but it already is used by the setting name.